dc-cudami-admin/dc-cudami-admin-webapp/src/main/java/de/digitalcollections/cudami/admin/controller/identifiable/entity/ContentTreesController.java

```diff
@@ -96,31 +94,25 @@ public String list(
   @PostMapping("/api/contenttrees/new")
   public ResponseEntity save(@RequestBody ContentTree contentTree)
       throws IdentifiableServiceException {
-    ContentTree contentTreeDb = null;
-    HttpHeaders headers = new HttpHeaders();
     try {
-      contentTreeDb = service.save(contentTree);
-      headers.setLocation(URI.create("/contenttrees/" + contentTreeDb.getUuid().toString()));
+      ContentTree contentTreeDb = service.save(contentTree);
+      return ResponseEntity.ok(contentTreeDb);
```

Comment:
nit: `.created(...)` wäre semantisch besser

Reply:
Ok, macht Sinn, aber wie ist es beim Update?

Reply:
da ist `.ok()` in ordnung :-) https://github.com/for-GET/http-decision-diagram

Reply:
:+1: